back/routes/adventure.js

```diff
@@ -24,4 +24,106 @@ router.post('/restaurants', async(req, res) => {
   });
 })
 
+Parse.initialize(process.env.NODE_ENV_ID_PROJECT, process.env.NODE_ENV_PROJECT_KEY);
+Parse.serverURL = 'http://parseapi.back4app.com';
+
+// ----- Get current user's preferences -----
+router.post('/preferences/current', async(req, res) => {
+  const query = new Parse.Query('UserPreference');
+  query.equalTo("username", req.body.username)
```

Comment:
I just tested that, and it seems like Parse takes care of that.
If user 'a' already exists, it doesn't allow a signUp under a username 'A'
However, for login in it would require the user to type 'a' and wouldn't recognize 'A'